tests/test_permissions_cli.py

```diff
@@ -0,0 +1,348 @@
+import responses
+
+from cap_client.utils import json_dumps
+
+
+# GET
+@responses.activate
+def test_permissions_get(cli_run):
+    responses.add(
+        responses.GET,
+        "https://analysispreservation-dev.cern.ch/api/deposits/some-pid",
+        json={
+            'permissions': {
+                'deposit-admin': {
+                    'roles': [1],
+                    'users': ['cms@inveniosoftware.org']
+                },
+                'deposit-read': {
+                    'roles': [],
+                    'users': ['cms@inveniosoftware.org']
+                },
+                'deposit-update': {
+                    'roles': [],
+                    'users': ['cms@inveniosoftware.org']
+                }
+            }
+        },
+        status=200)
+
+    res = cli_run("permissions get -p some-pid")
+
+    assert res.exit_code == 0
+    assert res.stripped_output == json_dumps({
+        'deposit-admin': {
+            'roles': [1],
+            'users': ['cms@inveniosoftware.org']
+        },
+        'deposit-read': {
+            'roles': [],
+            'users': ['cms@inveniosoftware.org']
+        },
+        'deposit-update': {
+            'roles': [],
+            'users': ['cms@inveniosoftware.org']
+        }
+    })
+
+
+def test_permissions_get_no_pid_given(cli_run):
+    res = cli_run("permissions get")
+
+    assert res.exit_code == 2
+    assert "Error: Missing option '--pid' / '-p'." in res.output
+
+
+@responses.activate
+def test_permissions_get_pid_not_exists(cli_run):
+    responses.add(
+        responses.GET,
+        'https://analysispreservation-dev.cern.ch/api/deposits/some-pid',
+        json={
+            'status': 404,
+            'message': 'PID does not exist.'
+        },
+        status=404)
+
+    res = cli_run("permissions get -p some-pid")
+
+    assert res.exit_code == 1
+    assert res.stripped_output == 'PID does not exist.'
+
+
+@responses.activate
+def test_permissions_get_no_access(cli_run):
+    responses.add(
+        responses.GET,
+        'https://analysispreservation-dev.cern.ch/api/deposits/some-pid',
+        json={
+            'message': "You don't have the permission to access the requested resource. "
+            "It is either read-protected or not readable by the server.",
+            'status': 403
+        },
+        status=403)
+
+    res = cli_run("permissions get -p some-pid")
+
+    assert res.exit_code == 1
+    assert res.stripped_output == "You don't have sufficient permissions."
+
+
+# ADD
+@responses.activate
+def test_permissions_add(cli_run):
```

Comment:
i know you know ^^  but i will just add here so we remember, that we should check if request was sent with proper headers and body, e.g.
```
    request = responses.calls[0].request
    assert request.headers['Accept'] == 'application/permissions+json'
    assert json.loads(request.body) == [{
        "email": "cms2@inveniosoftware.org",
        "type": "user",
        "op": "add",
        "action": "deposit-read"
    }, {
        "email": "cms2@inveniosoftware.org",
        "type": "user",
        "op": "add",
        "action": "deposit-update"
    }]
```
Those checks should be in all the tests that send some data to server or ask for some special content-type